clippy_utils/src/msrvs.rs

```diff
@@ -19,6 +19,7 @@ macro_rules! msrv_aliases {
 
 // names may refer to stabilized feature flags or library items
 msrv_aliases! {
+    1,70,0 { OPT_IS_SOME_AND }
```

Comment:
nit: We should probably name this `OPTION_IS_SOME_AND` for consistency with the other msrvs.